src/Symfony/Component/Lock/Store/MysqlStore.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Symfony\Component\Lock\Store;
+
+use Symfony\Component\Lock\Exception\LockConflictedException;
+use Symfony\Component\Lock\Key;
+use Symfony\Component\Lock\StoreInterface;
+
+class MysqlStore implements StoreInterface
```

Comment:
should be final

Reply:
Why not, but why ?

Reply:
We keep having this conversation again and again :)
We don't do final by default in Symfony, it would reduce the value of the code base with no real benefit for maintenance.

Reply:
On the contrary, it increases the value of the code base, because the developer can clearly see which are extension points and which are not. It will also force the developer to use composition and service decorators, because they can't get around with ugly hacks such as changing the service class.

It will actually make maintenance a lot easier because it's no longer an extension point 🤔 

> We don't do final by default in Symfony

Maybe we should ;)

Reply:
> We don't do final by default in Symfony

Wait, but it is not by default, it's by @iltar's request. :) I don't see valid use case to inherit `MysqlStore` as well. What's the point to support it? 🤔